src/components/home/DialogSelect.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import InputLabel from '@material-ui/core/InputLabel';
+import Input from '@material-ui/core/Input';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select'; import { FileCopy } from '@material-ui/icons';
+
+const styles = theme => ({
+    container: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+    formControl: {
+        margin: theme.spacing.unit,
+        minWidth: 120,
+    },
+});
+
+class DialogSelect extends React.Component {
+    state = {
+        open: false,
+        age: '',
+        back2Back: null,
+        noShow: null
+    };
+
+    handleChange = name => event => {
+        this.setState({ [name]: event.target.value });
+    };
+
+    handleClickOpen = () => {
+        this.setState({ open: true });
+    };
+
+    handleClose = () => {
+
+        this.setState({ open: false });
+        navigator.clipboard.writeText(`
```

Comment:
<img width="631" alt="screen shot 2018-12-26 at 12 31 57 pm" src="https://user-images.githubusercontent.com/16419909/50452802-5f050c00-090a-11e9-8126-9c4be0a39919.png">

The above code results in the following below. 
```
1.90e8468c-c77f-43ce
2.Antone Batz
3.B2B-Yes
4.YES
```

Suggestion:
How about we have it like 
```
1.Class Code: 90e8468c-c77f-43ce
2.Student Name: Antone Batz
3.B2B-Yes
4.No-Show: YES
```